traffic_ops/install/bin/postinstall.py

```diff
@@ -0,0 +1,1248 @@
+#!/usr/bin/python3
```

Comment:
If we want to support python2, too, this should be just `python`, not `python3`.

CentOS 7 and CentOS 8 both have a python interpreter at `/usr/libexec/platform-python` (on CentOS 7 it is Python 2.7, on CentOS 8, it is Python 3.6). To accept a binary named `python` if it exists but falls back to `platform-python`, the shebang could be something like

```shell
#!/usr/bin/env -S sh -c 'PATH="${PATH}:/usr/libexec" exec "$(type -p python platform-python | head -n1)" "$0" "$@"'
```